### PR TITLE
chore: set up cloudbuild file

### PR DESCRIPTION
**Pull request by peter-vizi-pager**
Have to make one small adjustment to how we create the default jacrkabbit instance: in cloudbuild Rabbit is listening on the `amqp://rabbitmq` address and not on `amqp://127.0.0.1` as it does when locally testing or apparently on circleci. We have to respect the `RABBIT_URL`  environment variable to connect to the "default" Rabbit instance. This is the same for how minion-army [handles](https://github.com/pagerinc/minion-army/blob/88d056f9025b35e59d897c1a4c0f42273e6078ea/lib/index.js#L53) the `RABBIT_URL` environment variable.

A little cleanup is done to how the debug library is used here, we don't have to `JSON.stringify` the every message received.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -87,9 +87,17 @@ const createService = (exchange, handler, options) => {
     });
 };
 
+const createJackrabbit = (options) => {
+    /* $lab:coverage:off$ */
+    const rabbitUrl = options.rabbitUrl || process.env.RABBIT_URL || 'amqp://127.0.0.1';
+    /* $lab:coverage:on$ */
+    Debug('creating Jackrabbit instance with url %s', rabbitUrl);
```
Comment (BlaineEhrhart) [inline lib/index.js:94]: Can't this leak the password for the rabbit connection?

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "@hapi/lab": "24.x",
     "@pager/eslint-config": "1.x",
     "@pager/semantic-release-config": "2.x",
-    "eslint": "8.x",
+    "eslint": "7.x",
```
Comment (BlaineEhrhart) [inline package.json:41]: Why downgrade?

**Review by BlaineEhrhart — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (peter-vizi-pager) [reply package.json:41]: At the moment `@pager/eslint-config` supports 7: https://github.com/pagerinc/eslint-config/blob/4bb49d68c1263e01139de8ca10c0ea7436dac23b/package.json#L23 I think Joan has startet looking into upgrading that but I'm not sure if that work is ready yet.

File: cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    name: node:18
+    entrypoint: npm
+    args: ['test']
+    env:
+      - 'NODE_ENV=test'
+      - 'RABBIT_URL=amqp://rabbitmq:5672'
+    waitFor: ['npm-ci', 'docker-compose']
+
+  - id: npm-publish
+    name: 'gcr.io/$PROJECT_ID/cloudbuilders/npm:6.13.4'
```
Comment (IJuanI) [inline cloudbuild.yaml:31]: This is an old version of npm, node v18.12.1 uses npm v8.19.2

**Review by BlaineEhrhart — APPROVED** [review]

**Pull request merged**